docs/team/yuxianglim.md

```diff
@@ -18,14 +18,23 @@ health data and weight goal.
 ### Code Contributed
 **__Diet Manager__**:<br>
 Link to code on tP Code Dashboard 
-[**__here__**](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#breakdown=true&search=lamyuewei&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos).
+[**__here__**](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#breakdown=true&search=yuxianglim&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos).
```

Comment:
Good job identifying the right name to use, ahahah

Reply:
Thanks haha i mean if i use yours i will get more code right ahaah